cellxgene_schema_cli/tests/fixtures/examples_validate.py

```diff
@@ -0,0 +1,205 @@
+import pandas as pd
+import numpy
+import anndata
+import os
+
+SCHEMA_VERSION = "2.0.0"
+FIXTURES_ROOT = os.path.join(os.path.dirname(__file__))
+
+# Pre-made example files
+h5ad_dir = os.path.join(FIXTURES_ROOT, "h5ads")
+h5ad_valid = os.path.join(h5ad_dir, "example_valid.h5ad")
+h5ad_invalid = [
+    "example_invalid_CL.h5ad",
+    "example_invalid_assay.h5ad",
+    "example_invalid_disease.h5ad",
+    "example_invalid_organism.h5ad",
+    "example_invalid_primary_data.h5ad",
+    "example_invalid_sex.h5ad",
+    "example_invalid_tissue.h5ad",
+    "example_invalid_ethnicity.h5ad",
+    "example_invalid_development_stage.h5ad",
+    "example_invalid_uns.h5ad",
+]
+h5ad_invalid = [os.path.join(h5ad_dir, i) for i in h5ad_invalid]
+
+
+# Manual minimal examples
+good_obs = pd.DataFrame(
+    [
+        [
+            "CL:0000066",
+            "EFO:0009899",
+            "MONDO:0100096",
+            "NCBITaxon:9606",
+            "PATO:0000383",
+            "UBERON:0002048",
+            True,
+            "HANCESTRO:0575",
+            "HsapDv:0000003",
+        ],
+        [
+            "CL:0000192",
+            "EFO:0010183 (sci-plex)",
+            "PATO:0000461",
+            "NCBITaxon:10090",
+            "unknown",
+            "CL:0000192 (cell culture)",
+            False,
+            "na",
+            "MmusDv:0000003",
+        ],
+    ],
+    index=["X", "Y"],
+    columns=[
+        "cell_type_ontology_term_id",
+        "assay_ontology_term_id",
+        "disease_ontology_term_id",
+        "organism_ontology_term_id",
+        "sex_ontology_term_id",
+        "tissue_ontology_term_id",
+        "is_primary_data",
+        "ethnicity_ontology_term_id",
+        "development_stage_ontology_term_id",
+    ],
+)
+
+obs_expected = pd.DataFrame(
```

Comment:
I'm confused by the `expected` term here, since it's not being used directly in an assertion; only used to created to init `adata_with_labels`, below. Does `expected` mean something not related to test assertions?